python_terraform/terraform.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         var_file: Optional[str] = None,
         terraform_bin_path: Optional[str] = None,
         is_env_vars_included: bool = True,
+        terraform_version: Optional[float] = 0.13
```

Comment:
Maybe we can find here sth better than float to encode terraform versions?
We will need comparison operators on the encoded versions.
Any idea?

Reply:
I've added a second comment at a location where we use the terraform version. There are many more locations like the commented one.

Reply:
We could eventually use [packaging.version](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python) ?
```
>>> from packaging import version
>>> version.parse("2.3.1") < version.parse("10.1.2")
True
```

Reply:
sounds good :-)

Reply:
Implemented your suggestion on our repo:
https://gitlab.com/domaindrivenarchitecture/python-terraform/-/merge_requests/2

What do you think ?

If  you agree I will sqash & merge. You can bring back my change by cherry pick?